nsis/SectionCoreReqs.nsh

```diff
@@ -53,20 +53,13 @@
     ; in order to run (including CDVD!) -- and really there should be more but we don't have working
     ; SPU2 null plugins right now.
  
-    File ..\bin\Plugins\GSnull.dll
+    ;File ..\bin\Plugins\GSnull.dll
```

Comment:
It might be of some use for debugging, but it doesn't seem like something the end user would need. I don't recall anyone complaining that they were removed from the v1.5.0 builds. Someone who would need it would likely be able to build it from source. It could also be made available in the plugins section of the website for the three people who do think they need it. :)